filter_metrics/filter_metrics.yaml

```diff
@@ -0,0 +1,222 @@
+        vxlan:
+          $ref: '../flow/packet-headers/vxlan.yaml#/components/schemas/Filter.Vxlan'
+          x-field-uid: 6
+        ipv4:
+          $ref: '../flow/packet-headers/ipv4.yaml#/components/schemas/Filter.Ipv4'
+          x-field-uid: 7
+        ipv6:
+          $ref: '../flow/packet-headers/ipv6.yaml#/components/schemas/Filter.Ipv6'
+          x-field-uid: 8
+        pfcpause:
+          $ref: '../flow/packet-headers/pfcpause.yaml#/components/schemas/Filter.PfcPause'
+          x-field-uid: 9
+        ethernetpause:
+          $ref: '../flow/packet-headers/ethernetpause.yaml#/components/schemas/Filter.EthernetPause'
+          x-field-uid: 10
+        tcp:
+          $ref: '../flow/packet-headers/tcp.yaml#/components/schemas/Filter.Tcp'
+          x-field-uid: 11
+        udp:
+          $ref: '../flow/packet-headers/udp.yaml#/components/schemas/Filter.Udp'
+          x-field-uid: 12
+        gre:
+          $ref: '../flow/packet-headers/gre.yaml#/components/schemas/Filter.Gre'
+          x-field-uid: 13
+        gtpv1:
+          $ref: '../flow/packet-headers/gtp.yaml#/components/schemas/Filter.Gtpv1'
+          x-field-uid: 14
+        gtpv2:
+          $ref: '../flow/packet-headers/gtpv2.yaml#/components/schemas/Filter.Gtpv2'
+          x-field-uid: 15
+        arp:
+          $ref: '../flow/packet-headers/arp.yaml#/components/schemas/Filter.Arp'
+          x-field-uid: 16
+        icmp:
+          $ref: '../flow/packet-headers/icmp.yaml#/components/schemas/Filter.Icmp'
+          x-field-uid: 17
+        icmpv6:
+          $ref: '../flow/packet-headers/icmpv6.yaml#/components/schemas/Filter.Icmpv6'
+          x-field-uid: 18
+        ppp:
+          $ref: '../flow/packet-headers/ppp.yaml#/components/schemas/Filter.Ppp'
+          x-field-uid: 19
+        igmpv1:
+          $ref: '../flow/packet-headers/igmpv1.yaml#/components/schemas/Filter.Igmpv1'
+          x-field-uid: 20
+        mpls:
+          $ref: '../flow/packet-headers/mpls.yaml#/components/schemas/Filter.Mpls'
+          x-field-uid: 21
+        snmpv2c:
+          $ref: '../flow/packet-headers/snmpv2c.yaml#/components/schemas/Filter.Snmpv2c'
+          x-field-uid: 22
+        rsvp:
+          $ref: '../flow/packet-headers/rsvp.yaml#/components/schemas/Filter.Rsvp'
+          x-field-uid: 23
+
+    Filter.Field.Custom:
+      type: object
+      properties:
+        offset:
+          type: integer
+          format: uint32
+          default: 0
+          x-field-uid: 1
+        value:
+          x-include: '#/components/schemas/Filter.Field/properties/value'
+          x-field-uid: 2
+        mask:
+          x-include: '#/components/schemas/Filter.Field/properties/mask'
+          x-field-uid: 3
+
+    Filter.Field:
+      type: object
+      properties:
+        value:
+          type: string
+          format: hex
+          default: "00"
+          x-field-uid: 1
+        mask:
+          type: string
+          format: hex
+          default: "00"
```

Comment:
default mask should be 0xff. But there could be problems with field width (i.e. field width varies with field). Perhaps we need a special variable-width hex value, e.g. 0xf..., or !0. If that won't work, we should not give mask a default value, because 00 is (almost?) never right.

Reply:
Removed default value for mask